### PR TITLE
Disables color output on Windows

### LINKED ISSUE #126: Console style characters appear in Windows as not understood
As seen in #125, seems that when colored/bold text should be inserted in the console, in Windows, unknown characters appear. Which means that in errors and in verbose outputs the readability is really bad.

This is due to our dependency in [`colored`](https://crates.io/crates/colored). As per their readme, they plan to support Windows consoles, but they still don't. An option is to disable coloring selectively in Windows until they implement it, and maybe even help them do it.

### PR DESCRIPTION
**Pull request by Cxarli**
Fixes #126

I have not tested this personally, but it should work as per [the cargo docs](http://doc.crates.io/specifying-dependencies.html#platform-specific-dependencies)

Happy Hacktoberfest!

**Note:** these two commits should be squashed into one

### CODE REVIEW

**Review by Razican — APPROVED** [review]
Thaank you for your contribution!

It looks good from my side, but don't have a Windows machine to check, so I will wait until @Chuky9 can review it in Windows to merge it.

**Comment by Razican** [comment]
I updated the base branch (pull request should be done to `develop`), and seems you should rebase, since some changes have created conflicts.

**Comment by Cxarli** [comment]
Should be fixed

File: Cargo.toml
Comment (Razican) [inline Cargo.toml]: This is making builds fail. It should be:

```toml
colored = { version = "^1.4", features = ["no-color"] }
```

**Review by Razican — CHANGES_REQUESTED** [review]
i, seems that the syntax in TOML is breaking the builds, could you rephrase?

Also, would it be possible to run `cargo update` in your branch so that changes get reflected in the `Cargo.lock` file?

**Comment by Cxarli** [comment]
Both fixed :+1:

File: Cargo.toml
Comment (Cxarli) [reply Cargo.toml]: Whoops, confused TOML with JSON there. :roll_eyes:

**Comment by Razican** [comment]
Seems that some builds are failing. It seems that the `Cargo.lock` file cannot be parsed with some features. Could you check if you can compile with different features?

**Comment by Razican** [comment]
I've been checking your branch, and the easy fix is to remove the Cargo.lock file and call `cargo update`. Then commit again.

**Review by Razican — APPROVED** [review]

**Review by Chuky9 — APPROVED** [review]
I have tested it in Windows and works perfectly despite Cargo.lock issues recently pointed out by Razican.

**Comment by Chuky9** [comment]
Merged! Thank you very much for your contribution!

**Comment by Cxarli** [comment]
You're welcome :smile: Is it right to assume that `Cargo.lock` is not an issue anymore?

**Comment by Razican** [comment]
I made some commits and fixed it ;)

**Pull request merged**
